casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -197,8 +195,12 @@ sealed abstract class MultiParentCasperInstances {
 
       def blockDag: F[BlockDag] = Capture[F].capture { _blockDag.get }
 
-      def tsCheckpoint(hash: ByteString): F[Option[Checkpoint]] = Capture[F].capture {
-        checkpoints.get.get(hash)
+      def storageContents(hash: StateHash): F[String] = Capture[F].capture {
+        if (knownStateHashesContainer.get.intersect(Set(hash)).isEmpty) {
```

Comment:
Apply method on sets determines if a value is contained: https://www.scala-lang.org/api/current/scala/collection/Set.html#apply(elem:A):Boolean

```scala
if (knownStateHashesContainer.get.apply(hash)) {
  runtimeManager.storageRepr(hash)
else {
  s"Tuplespace hash ${Base16.encode(hash.toByteArray)} not found!"
}